source/documentation/monitoring_apps/metrics.md

```diff
@@ -55,27 +57,31 @@ To set up the PaaS Prometheus exporter app:
 
 	You should use the `cf set-env` command for these mandatory variables as they contain secret information, and this method will keep them secure.
 
-	You can also set environment variables by amending the manifest file. We recommend that you use this method for optional environment variables that do not contain secret information. Refer to the [PaaS Prometheus exporter repository](https://github.com/alphagov/paas-prometheus-exporter) for more information.
+	You can also set environment variables by changing the manifest file. You should do this for optional environment variables that do not contain secret information. Refer to the [PaaS Prometheus exporter repository](https://github.com/alphagov/paas-prometheus-exporter) [external link] for more information.
 
-1. Configure your Prometheus service's metrics endpoint. The metrics endpoint is the deployed PaaS Prometheus exporter app URL with `/metrics` added on. For this example:
+1. Configure your Prometheus service's metrics endpoint. The metrics endpoint is the deployed PaaS Prometheus exporter app URL with `/metrics` added on to the end. For this example:
 
 	```
 	https://prometheus-exporter-exampleorg.cloudapps.digital/metrics
 	```
 
-1. Start the PaaS Prometheus exporter app by running `cf start prometheus-exporter`.
+1. Start the PaaS Prometheus exporter app:
+
+	```
+	cf start prometheus-exporter
+	```
 
 You can now check your Prometheus service to see if you are collecting metrics.
 
-If you are not receiving any metrics, check the PaaS Prometheus exporter app [logs](/monitoring_apps.html#logs). 
+If you do not receiving any metrics, check the PaaS Prometheus exporter app [logs](/monitoring_apps.html#logs).
 
-If you still need help, please contact us by emailing [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
+If you still need help, contact us by emailing [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
 
-#### Deploy the app with authentication
+#### Add authentication
```

Comment:
There is a PR in the paas-prometheus-exporter that would render this comment obsolete: https://github.com/alphagov/paas-prometheus-exporter/pull/16

Reply:
Should I raise a PR to delete this section after PR #16 is merged?